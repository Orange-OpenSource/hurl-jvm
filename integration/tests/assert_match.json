{"entries":[{"request":{"method":"GET","url":"http://localhost:8000/assert-match"},"response":{"version":"HTTP/1.0","status":200}}]}
