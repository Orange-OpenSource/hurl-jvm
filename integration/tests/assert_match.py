from tests import app
from flask import Response

@app.route("/assert-match")
def assert_match():
    return Response('''{
  "date1": "2014-01-01",
  "date2": "x2014-01-01"
}''', mimetype='application/json')


