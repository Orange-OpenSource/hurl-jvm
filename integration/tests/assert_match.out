{
  "date1": "2014-01-01",
  "date2": "x2014-01-01"
}